frontend/server/src/DAO/Contests.php

```diff
@@ -425,7 +426,8 @@ final public static function getContestsParticipating(
             SELECT
                 $columns,
                 p.scoreboard_url,
-                p.scoreboard_url_admin
+                p.scoreboard_url_admin,
+                count(Problemset_Identities.identity_id) AS `contestants`
```

Comment:
```suggestion
                COUNT(Problemset_Identities.identity_id) AS `contestants`
```

intentamos que todas las palabras reservadas vayan en mayúsculas. igual en las demás consultas de este cambio.

Reply:
Corregido :) y **ya agregué las pruebas**, **OJO**, las 4 que agregué eran necesarias para testear las subqueries desde los puntos más importantes que pueden llegar a ser llamadas y utilizadas.